lib/utils.js

```diff
@@ -94,7 +94,9 @@ const redactUrlMatcher = (...matchers) => {
       try {
         const url = new URL(subValue)
         return redactMatchers(...matchers)(subValue, { url })
+      /* istanbul ignore next */
```

Comment:
In cli land if we ignore coverage we give a comment explaining why it can't be tested to show that we at least know what that line does and when it would be hit.

If we can't actually hit it, we may not need it.

Reply:
I removed this line, and broke the function out. I want to keep the try / catch I think it's important to ensure the url is a url without throwing even because trusting the regex is not great.